directory.md

```diff
@@ -154,6 +154,10 @@ Långt ifrån komplett! [Lägg gärna till](https://github.com/rails-se/rails-se
   <li>
     <a href="http://www.promoterapp.com">Promoter</a>
   </li>
+  
+  <li>
+    <a href="https://www.shopify.com/careers">Shopify</a>
```

Comment:
I've made this point before and I'll re-echo it.

We've tried not to include career pages in these links as this page is about creating visibility about which companies are out here in Stockholm rather than advertising about that they're hiring.

Reply:
I removed the other entry in https://github.com/rails-se/rails-se.github.com/pull/46

Reply:
Why not let everyone have a careers link?

Reply:
I don't think it's the right forum to advertise that a company is hiring for a website listing the companies that work with a given technology. If someone is interested in that company and what they do, they are welcome to find their career page on their own.